x-pack/plugins/fleet/public/components/with_guided_onboarding_tour.tsx

```diff
@@ -95,11 +95,16 @@ export const WithGuidedOnboardingTour: FunctionComponent<{
       title={config.title}
       anchorPosition={tourPosition ? tourPosition : 'rightUp'}
       footerAction={
-        <EuiButton onClick={() => setIsGuidedOnboardingTourOpen(false)} size="s" color="success">
+        <EuiButtonEmpty
+          size="xs"
```

Comment:
Followed the styles from @alisonelizabeth's [PR](https://github.com/elastic/kibana/pull/147636/files#diff-3249d6e800c47e67906db2807bd5814627dd6d4b71e8ebdb6d93881260b37dd6R100-R102)